x/cronos/keeper/permissions.go

```diff
@@ -0,0 +1,32 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/crypto-org-chain/cronos/x/cronos/types"
+)
+
+const (
+	CanChangeTokenMapping uint64                                  = 1 << iota // 1
+	CanTurnBridge                                                             // 2
+	All                   = CanChangeTokenMapping | CanTurnBridge             // 3
+)
+
+func (k Keeper) SetPermissions(ctx sdk.Context, address sdk.AccAddress, permissions uint64) {
+	store := ctx.KVStore(k.storeKey)
+	permissionsBytes := sdk.Uint64ToBigEndian(permissions)
```

Comment:
any reason why big endian? or it's a common convention in SDK to store integers like that?

Reply:
mmm not sure if it is generally adopted convention but ethermint uses this format to store uint64.

Reply:
in key it's usually encoded as big endian so it's same order as the number, but for value it should be irrelevant, I'm not sure if golang do optimization for little endian though.